src/ensembl/src/content/app/browser/browser-nav/BrowserNavBarMain.scss

```diff
@@ -0,0 +1,41 @@
+@import 'src/styles/common';
+
+.browserNavBarMain {
+  padding-left: 50px;
+  display: flex;
+  align-items: center;
+  flex-grow: 1;
+}
+
+.content {
+  display: flex;
+  flex-grow: 1;
+}
+
+.contentChromosomeNavigator {
+  flex-grow: 1;
+  max-width: 1400px;
+  margin: auto;
+}
+
+.contentSwitcherArea {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  flex: 0 0 auto;
+  width: 100px;
+}
+
+.contentSwitcher {
+  display: inline-block;
+  color: $blue;
+  cursor: pointer;
+
+  svg {
+    height: 20px;
+  }
+}
+
+.contentSwitcherClose {
+  font-size: 0;
```

Comment:
Can we use something like `line-height: 10px;` instead of `font-size` as there is no text?

Reply:
I would argue that `font-size` is better: it removes all `line-height`-based calculations from an inline-block element, and provides the developer with a container that has a predictable height, which is controlled solely by the height of its content.

Reply:
I think it would be nice to add what you just explained as a comment above the class just to make it very clear. I am sure this is not something we can easily infer just by looking at `font-size: 0;`.

Reply:
Added a comment, but it's generally a fairly common CSS technique, and not for the first time used in our codebase.